pruner.sh

```diff
@@ -17,8 +17,14 @@ while true; do
   #
   # https://bugzilla.redhat.com/show_bug.cgi?id=1408676
   if [ $(date +%H) -eq 3 ]; then
-    echo "Pruning old images from registry"
-    oc adm prune images --confirm
+    # Check if insecure registry variable is set
+    if [ $INSECURE_REGISTRY ]; then
```

Comment:
This should be rewritten to `if [ "$INSECURE_REGISTRY" = true ]; then`. Otherwise all values will evaluate to true. Also see: https://stackoverflow.com/questions/2953646/how-to-declare-and-use-boolean-variables-in-shell-script